samples/net/coap_client/README.rst

```diff
@@ -122,18 +122,10 @@ References
 Dependencies
 ************
 
-This sample uses the following |NCS| library:
-
-* :ref:`lte_lc_readme`
-
-It uses the following `sdk-nrfxlib`_ library:
-
-* :ref:`nrfxlib:nrf_modem`
-
-It uses the following Zephyr library:
-
-* :ref:`CoAP <zephyr:networking_api>`
-
-In addition, it uses the following secure firmware component:
+This sample uses the following Zephyr libraries:
 
-* :ref:`Trusted Firmware-M <ug_tfm>`
+* :ref:`net_if_interface`
+* :ref:`net_mgmt_interface`
+* :ref:`CoAP client <zephyr:coap_client_interface>`
+* :ref:`CoAP <zephyr:coap_sock_interface>`
+* :ref:`Connection manager <zephyr:conn_mgr_overview>`
```

Comment:
```suggestion
* :ref:`Connection Manager <zephyr:conn_mgr_overview>`
```

Reply:
Discussed offline, it can be taken care by @divipillai in another PR. Approving.